Doc/library/weakref.rst

```diff
@@ -201,6 +201,8 @@ than needed.
 same issues as the :meth:`keyrefs` method of :class:`WeakKeyDictionary`
 objects.
 
+   .. versionchanged:: 3.9
+      Added support for ``|`` and ``|=`` operators, specified in :pep:`584`.
```

Comment:
I think this is better moved up to line 174, just after the `..note` (and at the same indentation level).

Reply:
I think it should be more like line 200, because I believe line 174 is under `WeakKeyDictionary` rather than `WeakValueDictionary`?

Reply:
Yep, nice catch.